src/index.js

```diff
@@ -1,10 +1,26 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
+import 'semantic-ui-css/semantic.min.css';
```

Comment:
Nice :) Have you looked at ant.design ?

Reply:
Nope, Would do that but would also want you to tell us a bit about during today's standup

Reply:
Will do :)